prov/gni/test/rdm_multi_recv.c

```diff
@@ -890,7 +890,7 @@ void do_multirecv_trunc_last(int len)
  * message size of 1 below might change depending on whether 0 is a
  * valid value for FI_OPT_MIN_MULTI_RECV (Github issue #1120)
  */
-Test(rdm_multi_r, multirecv_trunc_last, .disabled = true)
+Test(rdm_multi_r, multirecv_trunc_last, .disabled = false)
 {
```

Comment:
Could you enable tests multirecv, multirecv_retrans, and multirecv_trunc_last_retrans in this file as well?